systems/libseff/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:18.04
```

Comment:
We should probably allow using more modern versions.

Reply:
The only reason I didn't was because we need clang-10 to compile libseff (it is specified in its Makefile), and clang-10 is not available in the latest ubuntu version as an apt package or from the llvm install script.

Reply:
Ok, so it compiles fine with `clang-14`. I will upgrade the clang version and the Ubuntu version to 22.04.